sms-notifier/LambdaFunction.js

```diff
@@ -2,30 +2,28 @@
 var AWS = require('aws-sdk');
```

Comment:
pputhran: All the var(s) can be replaced with let as I see we are using ‘const’ if we want to strictly follow ES6